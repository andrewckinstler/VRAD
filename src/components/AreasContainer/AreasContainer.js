import React from 'react';
import './AreasContainer.scss';
import { AreaCard } from '../AreaCard/AreaCard'

export const AreasContainer = (props) => {
  const areas = props.areas.map(area => <AreaCard {...area} key={area.id}/>)
  return(
    <div className='area-container'>
      {areas}
    </div>
  )
}