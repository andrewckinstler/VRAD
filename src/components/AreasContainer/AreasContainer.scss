.area-container{
  display: flex;
  flex-flow: row-nowrap;
  padding: 40px 35px;
}
