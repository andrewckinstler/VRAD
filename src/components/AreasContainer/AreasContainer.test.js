import React from 'react';
import { shallow } from 'enzyme';
import { AreasContainer } from './AreasContainer';

describe('AreasContainer', () => {
  it('should match the snapshot with all data passed in correctly', () => {
    let areas = [{
      nickname: 'test nickname',
      longname: 'test longname',
      description: 'test description',
      id: 5
    }];
    const wrapper = shallow(<AreasContainer
      areas={areas}
    />);
  expect(wrapper).toMatchSnapshot();
  });
});
