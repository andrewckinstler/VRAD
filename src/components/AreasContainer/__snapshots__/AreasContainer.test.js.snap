// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AreasContainer should match the snapshot with all data passed in correctly 1`] = `
<div
  className="area-container"
>
  <AreaCard
    description="test description"
    id={5}
    key="5"
    longname="test longname"
    nickname="test nickname"
  />
</div>
`;
